File: Nuova cartella/c/acceso_spento.c
#include <stdio.h>
#include <stdlib.h>

//Compiler version gcc  6.3.0

int main()
{
  int temperatura_interna, temperatura_esterna, umidità, acceso, spento;
  printf("temperatura interna");
  scanf("%d",& temperatura_interna);
  printf("temperatura esterna");
  scanf("%d",& temperatura_esterna);
  printf("umidità");
  scanf("%d",& umidità);
  
  if(temperatura_interna>70)
  {
    printf("acceso");
  }
  else
  if(temperatura_esterna>100)
  {
    printf("acceso");
  }
  else
  if(umidità>90)
  {
    printf("acceso");
  }
  else
  printf("spento");
  return 0;
}

File: Nuova cartella/py/acceso_spento.py
#python 2.7.15
temperatura interna=int(input("temperaturainterna:"))
temperatura esterna=(input("temperaturaesterna:"))
umiditá=int(input("umidità"))
if(temperatura interna>70):
print("acceso")
else:
  if(temperatura esterna>100):
    print("acceso")
    else:
      if(umidità>90):
      print("acceso")
      else:
        print("spento")